components/proposal/Card.vue

```diff
@@ -198,6 +218,10 @@ watch(reasonForVote, (value) => {
   emit("update-reason-for-vote", value, props.proposal.proposalId);
 });
 
+const errorProposal = computed(() => {
+  return errorProposals?.some((id) => id === Number(props.proposal.proposalId));
```

Comment:
```suggestion
  return errorProposals?.some((id) => id === props.proposal.proposalId);
```
if is stored as string